scripts/prepare_kb.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
Move this scripts to kb_scripts folder

Reply:
This script should be called in build_kb.sh if your pass to this script key "-p".
build_kb.sh should process prepared kb